src/dune

```diff
@@ -4,47 +4,28 @@
  (modules CryptokitBignum Cryptokit)
  (foreign_stubs
   (language c)
-  (flags -DCAML_NAME_SPACE
+  (flags -DCAML_NAME_SPACE -DEXPORT=static
          -DBLAKE3_NO_SSE2 -DBLAKE3_NO_SSE41 -DBLAKE3_NO_AVX2 -DBLAKE3_NO_AVX512
          -DBLAKE3_USE_NEON=0
          (:include flags.sexp))
-  (names aesni
-         arcfour
```

Comment:
removing these names from the dune file makes them unavailable in dune's `_build` directory, leading to the failures you're seeing in https://github.com/ocaml/opam-repository/pull/26209

Reply:
It works just fine locally when I do `dune build`, so I don't understand what OPAM does that causes the build to fail.  Any idea?

At any rate, these .c files are used as include files, so I don't want Dune to build them and include them in the library.

Reply:
does it also work locally if you run `dune clean && dune build -p cryptokit`? 

I have 2 hypotheses:

1. you had a stale build so those files were already in the build directory
2. dune only filters those files out in the release archive

Reply:
It works with `dune build` but not with `dune build -p cryptokit`.  I miss the old Makefile...

Reply:
BTW, what do you mean by "the release archive"?  These .c files are sources used at build-time but are not part of what's installed.  I'm totally confused :-(

Reply:
I think we invalidated hypothesis 2, and I'm not sure it would hold either

Reply:
@xavierleroy I'm working on a fix, will send a PR shortly

Reply:
check https://github.com/xavierleroy/cryptokit/pull/37